src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -28,12 +28,12 @@
         -error   -->
     <!-- The default level of a check is "error". -->
     <!-- See http://checkstyle.sourceforge.net/config.html#Severity -->
-    <property name="severity" value="info" />
-    <property name="ignoreLines" value="1,2,3"/>
+    <property name="severity" value="error" />
+    <property name="ignoreLines" value="2"/>
     <property name="fileExtensions" value="java"/>
     <property name="header"
         value="${checkstyle.headerCheck.content}"
-        default="/**\n * Copyright (c) 2010-year by the respective copyright holders.\n *\n * All rights reserved. This program and the accompanying materials\n * are made available under the terms of the Eclipse Public License v1.0\n * which accompanies this distribution, and is available at\n * http://www.eclipse.org/legal/epl-v10.html\n */"/>  </module>
+        default="/**\n * Copyright (c) 2010-2017 by the respective copyright holders.\n *\n * All rights reserved. This program and the accompanying materials\n * are made available under the terms of the Eclipse Public License v1.0\n * which accompanies this distribution, and is available at\n * http://www.eclipse.org/legal/epl-v10.html\n */"/>  </module>
```

Comment:
Can't we make the end year dynamic? Maybe by string replacing it `${current_year}` => _some java code_